app/src/main/java/org/holtnet/officialimperialbusiness/ExamActivity.java

```diff
@@ -121,30 +138,37 @@ public void onQuestionsRadioButtonClicked(View view) {
         }
     }
 
+
+    // if you're leveraging the onClick, remove them from the styles so the function name isn't grey below.
     public void onQuestionsCheckboxClicked(View view) {
         // Verify which checkboxes are checked and which are not, assigning a true or false value to the correct variable based on which question is being checked.
 
+        // This whole switch statement runs for every tick.  Did you forget to add breaks after each section?
```

Comment:
Yes: the intention is to determine whether q11 is true or false. In order to check if the proper check boxes arechecked or unchecked, All of the checkboxes need to be passed down to the end of that block of answers (i.e., the destiny checkbox). Depending on the configuration of the checkboxes (and allowing for continual changes to them) the q+number will be updated to true or false every time a box is checked or unchecked.

Reply:
You could just check everything once on submit instead.  Right now you're doing a check for every click.  

Furthermore, you didn't have the break statements in your code.  So when you ticked off a q11 box, it was also checking q12, 13 and 14 scenarios.  i.e. your switch statement was falling through.